### DISCUSSION TITLE
Require cycle error

### DISCUSSION THREAD

**Discussion by hanigm** [jpudysz/react-native-unistyles#251] [Q&A]
Hi there,

I have just started a new React native project and I wanted to use Unistyles for styling. However, when I open an Expo development build, I get the following and the app defaults to the +notFound page
![image](https://github.com/user-attachments/assets/5aafae88-21cf-40cf-9143-5c03c8cf2609)

**Comment by jpudysz** [comment]
Fixed in 2.9.0
